onisite/settings_base.py

```diff
@@ -207,7 +207,7 @@
 
 SOLR_BASE_URL = os.getenv('ONI_SOLR_URL', 'http://solr:8983')
 
-SOLR_CLOUD = False
+SOLR_CLOUD = os.getenv('ONI_SOLR_CLOUD', False)
```

Comment:
Sorry, I didn't catch this earlier. For bools, we use `0` and `1` values everywhere else in the config to bypass the casting of strings etc. You can see other examples in the documentation https://github.com/open-oni/open-oni/blob/dev/docs/customization/configuration.md

Reply:
oh, great. that certainly simplifies things.